sys/libraries/deployment_settings.php

```diff
@@ -0,0 +1,22 @@
+<?php
+class deployment_settings extends Library {
+
+    // Uncomment the next line to deactivate Missing Person Module
+    // private $missing_person = false;
+    
+    function __construct() {
+    }
+
+    private function get_settings($var_name, $default_value) {
+        if (isset($this->$var_name)) {
+            return $this->$var_name;
+        } else {
+            return $default_value;
+        }
+    }
+
+    function missing_person() {
```

Comment:
But this will lead to problems. I know it can be a headache initially, but this way will later on be easy to maintain. Sahana also follows this concept. I say we stick with it.